FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Hai Phung on 6/15/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it from the `ManagedCache` declaration, as well as from its properties and methods.
Also, this class is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class ManagedCache: NSManagedObject {
```